app/views/posts/index.html.erb

```diff
@@ -1 +1,54 @@
-<h1>show a list of posts</h1>
\ No newline at end of file
+<div class = 'users-tab'>
+  <h1>All Posts</h1>
+  <div>
+    <div class='users-header'>
+      <div class = 'img-container'>
+            <div class="inner-border">
+              <img src="<%= @posts.first.author.photo %>" alt="" width="100" height="100">
+            </div>
+      <%= link_to @posts.first.author.name, user_path(@posts.first.author) %>
+      </div>  
+        <div>
+          <h3>Posts: <%= @posts.first.author.post_counter %></h3>
+        </div>
+    </div>
+    <div class='posts-list'>
+    <% @posts.each do |post| %>
+      <div class="posts-index">
+        <div class="posts-title">
+          <h3>
+            <%# author id: <%= post.author_id %>
+            <%= link_to post.title, user_post_path(post.author_id, post.id) %> 
+          </h3>
+        </div>
+        <div class="posts-comment">
+          <h4>
+            <% if post.comments.empty? %> 
+              No comments yet
+            <% else %>
+              <ul>
+                <% post.comments.each do |comment| %>
+              </ul>
+              <li>
+                <%= comment.author.name %>:
+                <%= comment.text %>
+              </li>              
+                <% end %>
+            <% end %>
+          </h4>
+        </div>
+        <div>
+          <h4> comments: <%= post.comment_counter %> likes: <%= post.likes_counter %></h4>
+        </div>  
+      </div>
+    <% end %>
+    </div>
+
+  </div>

```

Comment:
- [ ] Congratulations on your effort on the additional two improvements that the prior reviewer asked :clap: . However I believe he is also correct in asking that you make sure that the '`users/ser/posts` page only displays the most recent 5 comments for each post.
Although you are partially correct that it is not stated in the requirements, the exact same thing can be seen when looking at the wireframe for the route `users/user/posts`.  And because we must adhere to the wireframe, please make sure to include that update. 🙏

### the wireframe in question :point_down: 

![Screenshot from 2023-09-29 16-11-03](https://github.com/Zven94/BlogApp/assets/89228743/40560e7f-921a-44ef-8473-8ebaebec1bad)

To complete that task the only thing you should do is update your code as follows:point_down: 
```suggestion
<div class = 'users-tab'>
  <h1>All Posts</h1>
  <div>
    <div class='users-header'>
      <div class = 'img-container'>
            <div class="inner-border">
              <img src="<%= @posts.first.author.photo %>" alt="" width="100" height="100">
            </div>
      <%= link_to @posts.first.author.name, user_path(@posts.first.author) %>
      </div>
        <div>
          <h3>Posts: <%= @posts.first.author.post_counter %></h3>
        </div>
    </div>
    <div class='posts-list'>
    <% @posts.each do |post| %>
      <div class="posts-index">
        <div class="posts-title">
          <h3>
            <%# author id: <%= post.author_id %>
            <%= link_to post.title, user_post_path(post.author_id, post.id) %>
          </h3>
          <p><%= post.text %></p>
        </div>
        <div class="posts-comment">
          <h4>
            <% if post.comments.empty? %>
              No comments yet
            <% else %>
              <ul>
                <% post.return_most_recent_comment.each do |comment| %>
              </ul>
              <li>
                <%= comment.author.name %>:
                <%= comment.text %>
              </li>
                <% end %>
            <% end %>
          </h4>
        </div>
        <div>
          <h4> comments: <%= post.comment_counter %> likes: <%= post.likes_counter %></h4>
        </div>
      </div>
    <% end %>
    </div>

  </div>

```
### What I did here is:
Instead of looping through `post.comments`, I looped through `post.return_most_recent_comment` in this update, and I just displayed the post content as it appeared in the wireframe.😊

This update is going to give a similar result compared to the wireframe. 👍️

### Result :point_down: 
![Screenshot from 2023-09-29 16-18-43](https://github.com/Zven94/BlogApp/assets/89228743/069abd55-977d-40d3-9152-e5869bb77b05)